src/pyfmi/fmi3.pyx

```diff
@@ -223,9 +266,27 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
         if self._log_stream:
             self._log_stream = None
 
+    def reset(self):
+        """ TODO """
+        self._t = None
```

Comment:
I left the reset function like this since it is to be implemented later. But I dont want us to forget about resetting `self._t` which I had to introduce in this PR.

Reply:
Fixed